src/fixtureParser.ts

```diff
@@ -77,22 +100,40 @@ export const parsePytestOutputToFixtures = (output: string, rootDir: string) =>
             }
             const [name, linePath, line] = matches.slice(1);
             fixture = { name, docstring: "" };
-
-            const path = isAbsolute(linePath) ? linePath : join(rootDirForPath, linePath);
+            let path;
+            // check if pytest fixture
+            if (linePath.indexOf(".../_pytest") > -1) {
```

Comment:
I think for here and line 116 it assumes the path separator is `/`, but it's `\` on windows. You can use [path.sep](https://nodejs.org/api/path.html#pathsep) instead